addons/project/models/project.py

```diff
@@ -437,7 +437,7 @@ def _inverse_partner_phone(self):
                 project.partner_id.phone = project.partner_phone
 
     @api.onchange('alias_enabled')
-    def _onchange_alias_name(self):
+    def _onchange_alias_enabled(self):
```

Comment:
This is the opposite, the `onchange` are named according to the field which value has to be changed, not according to the field(s) that triggers(trigger) the `onchange`.
The reason behind is to prevent having to create two `onchange` with the same code as you did.

Reply:
humm, are you sure ? 
![image](https://user-images.githubusercontent.com/64937092/200380372-306f2513-e0b7-4e39-abbe-37b0183762b9.png)
![image](https://user-images.githubusercontent.com/64937092/200380424-bc39a796-1604-4161-8291-03608ae049e3.png)
i searched for many examples in the code
same in the documentation
![image](https://user-images.githubusercontent.com/64937092/200380809-c3300d15-8a76-4373-a476-ca7d62725ac0.png)

Reply:
hum indeed. However I don't think that it is especially a good idea, I have to say, as when you will have to add a field in the @api.onchange(''), what will you do? How will you name the def? Will you keep the original name which will not mean anything anymore? But do the way you want.